packages/luajit.rb

```diff
@@ -27,7 +27,9 @@ def self.build
   def self.install
     system 'make', "PREFIX=#{CREW_PREFIX}", "MULTILIB=#{ARCH_LIB}", "DESTDIR=#{CREW_DEST_DIR}", 'install'
     Dir.chdir("#{CREW_DEST_PREFIX}/bin") do
-      FileUtils.ln_s 'luajit-2.1.0-beta3', 'luajit'
+      Dir["luajit-#{version.split('-').first}*"].each do |f|
+        FileUtils.ln_sf f, 'luajit'
```

Comment:
Don't you want a different target file each loop iteration?

Reply:
It's only one file.

Reply:
Good point.

Rectified.